web/packages/operations/src/alarm.ts

```diff
@@ -117,21 +165,17 @@ export const sendMetrics = async (metrics: status.AllMetrics) => {
         metricData.push({
             MetricName: AlarmReason.ToEthereumChannelStale.toString(),
             Value: Number(
-                    (channel.toEthereum.outbound > channel.toEthereum.inbound &&
-                        channel.toEthereum.inbound == channel.toEthereum.previousInbound)
+                channel.toEthereum.outbound > channel.toEthereum.inbound &&
+                    channel.toEthereum.inbound == channel.toEthereum.previousInbound
             ),
         })
         metricData.push({
             MetricName: AlarmReason.ToEthereumChannelAttacked.toString(),
-            Value: Number(
-                channel.toEthereum.outbound < channel.toEthereum.inbound     
-            ),
+            Value: Number(channel.toEthereum.outbound < channel.toEthereum.inbound),
```

Comment:
Instead of logging booleans, maybe we should log the difference. This would then be a metric of undelivered messages.

Reply:
https://github.com/Snowfork/snowbridge/pull/1336/commits/785b59ba70fde21abf0301b937dacc61e4a02e1a metrics for the difference.